server/src/main/java/org/candlepin/guice/CandlepinModule.java

```diff
@@ -148,6 +137,24 @@
 import com.google.inject.name.Names;
 import com.google.inject.persist.jpa.JpaPersistModule;
 
+import org.hibernate.cfg.beanvalidation.BeanValidationEventListener;
+import org.hibernate.validator.HibernateValidator;
+import org.hibernate.validator.HibernateValidatorConfiguration;
+import org.quartz.JobListener;
+import org.quartz.spi.JobFactory;
+import org.xnap.commons.i18n.I18n;
+
+import java.util.Properties;
```

Comment:
Please correct the automatic import ordering in your IDE.  If you are using Eclipse, you can go to Preferences, Java, Code Style, Organize Imports.  Then click the "Import..." button and select the file `project_conf/eclipse/candlepin.importorder`.

Using the same import order as everyone else will reduce superfluous changes in the diffs.